.openpublishing.redirection.json

```
@@ -4,6 +4,11 @@
             "redirect_url": "https://docs.microsoft.com/powershell/scripting/overview",
             "redirect_document_id": true
         },
+        {
+            "source_path": "reference/digital-art.md",
```

Need to add another record for 
```suggestion
            "source_path": "digital-art.md",
```